packages/apollos-data-connector-rock/src/features/__tests__/index.js

```diff
@@ -342,6 +342,26 @@ describe('features', () => {
       expect(await expandResult(result)).toMatchSnapshot();
     });
 
+    it('should create a PrayerListFeature from a DAILY_PRAYER algorithm', async () => {
+      const feature = new Feature();
+      feature.initialize({
+        context,
+      });
+
+      const result = await feature.createPrayerListFeature({
+        algorithms: [
+          {
+            type: 'DAILY_PRAYER',
+          },
+        ],
+        title: 'Test Action List',
+        subtitle: "It's great!",
+      });
+
+      expect(await expandResult(result)).toMatchSnapshot();
```

Comment:
In order for your snapshot to show up perfectly (it's 99% there) you'll need to make sure that the `expandResult` function expands the `prayer` key :)

Reply:
Help me understand what that looks like:

```suggestion
      expect(await expandResult(result.prayer)).toMatchSnapshot();
```

Reply:
https://github.com/ApollosProject/apollos-apps/blob/4df0de5c38bd9dcb96737b497458e3c35f85e43c/packages/apollos-data-connector-rock/src/features/__tests__/index.js#L17

It's actually up at the top, you need to add something like 

```
if result.prayer && prayer is a function 
  return the data, with prayerexpanded
end
```